<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-FF1NHT0" ignored="1" name="Default test(failed)" tests="2" failures="2" timestamp="12 May 2020 09:43:02 GMT" time="0.09" errors="0">
  <testcase name="tes3" time="0.019" classname="org.testprallel.RerunFailed">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testprallel.RerunFailed.tes3(RerunFailed.java:14)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tes3 -->
  <testcase name="test2" time="0.01" classname="org.testprallel.RerunFailed">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testprallel.RerunFailed.test2(RerunFailed.java:20)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test1" time="0.0" classname="org.testprallel.RerunFailed">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
